libkbfs/bserver_remote.go

```diff
@@ -264,13 +264,16 @@ func (b *BlockServerRemote) Put(ctx context.Context, id BlockID, tlfID TlfID,
 		// BlockKey is misnamed -- it contains just the server
 		// half.
 		BlockKey: serverHalf.String(),
-		Folder:   tlfID.String(),
+		Folder:   md.ID.String(),
 		Buf:      buf,
 	}
 
 	err = b.client.PutBlock(ctx, arg)
 	if err != nil {
 		if qe, ok := err.(BServerErrorOverQuota); ok && !qe.Throttled {
+			name := md.GetTlfHandle().GetCanonicalName()
```

Comment:
Hmm, I see.

What about just having `BlockServerRemote` pass the error up, and do the quota error checking and reporting in `BlockOps`? That seems like a better place anyway, since it's plausible that the local BlockServer implementations could be used for testing quota conditions.